docs/CONFIG-PROPERTIES.md

```diff
@@ -68,10 +68,11 @@
 | goroutine.leak.detection.cooldown.minutes | integer (minutes) | 5 | Cooldown period in minutes after the leak detection is triggered. During this period, no stack traces are collected; only warning messages are logged. |
 | kubevirt.drain.timeout | integer | 24 | hours to allow kubernetes to drain a node |
 | memory-monitor.enabled | boolean | false | Enable external memory monitoring and memory pressure events handling |
+| debug.tui.loglevel | string | info | Set log level for EVE Text UI (TUI) monitor. Possible values are "OFF", "ERROR", "WARN", "INFO", "DEBUG", "TRACE" and are case insensitive
 
 ## Log levels
 
-Log level can be set for three different components of EVE: EVE microservices, syslog, and kernel.
+Log level can be set for four different components of EVE: EVE microservices, syslog, kernel, and TUI monitor application. Logs for TUI monitor are not sent to the controller and only available locally on the device.
```

Comment:
Any reason to not send the TUI logs tp the controller?

Reply:
@eriknordmark there are two

1. It is difficult to do technically. I either need a client for memlogd or route all logs through pillar. I'm planing to solve this later
2. TUI  functionality should not affect eve. in the worst case it will be difficult to debug TUI but collect-info.sh solves it

Reply:
Ok